internal/c4/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/ThreeDotsLabs/wild-workouts-go-ddd-example/internal/c4
```

Comment:
Thanks for contributing!

Perhaps we could create a top-level `tools` directory and keep `c4` there? To not confuse people looking for services or application code in `internal`. 🙂

Reply:
https://github.com/ThreeDotsLabs/wild-workouts-go-ddd-example/pull/36/commits/ea2032e34a0adfeb62f79c1b550e7be2fb8a4cba